python/Dockerfile

```
@@ -3,4 +3,4 @@ FROM python:3.10-slim
 WORKDIR /app
 COPY . .
 
-ENTRYPOINT [ "python", "main.py" ]
\ No newline at end of file
+ENTRYPOINT [ "python", "deez_py" ]
```

```suggestion
ENTRYPOINT [ "python", "-m", "deez_py" ]
```
ModuleNotFoundError when running inside docker.